composer.json

```diff
@@ -25,7 +25,7 @@
     "lucatume/wp-snaphot-assertions": "1.1.0",
     "mikey179/vfsstream": "^1.6",
     "moderntribe/tribalscents": "dev-master",
-    "moderntribe/tribe-testing-facilities": "dev-master",
+    "moderntribe/tribe-testing-facilities": "dev-fix/views-v2-better-reset",
```

Comment:
Do you want to merge the other PR first and change this back?

Reply:
yes, I've updated in lates commit.